src/app/styles/css/mixins/variables.css

```diff
@@ -2,17 +2,29 @@
 :root {
   --columnWidthMedium: 680px;
   --columnWidthLarge: 880px;
-  --fontStackSans: 'Roboto', -apple-system, blinkmacsystemfont, 'Corbel', 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', verdana, 'Verdana Ref', sans-serif;
+  --font-family: inter var;
+  --fontStackSans: var(--font-family), -apple-system, blinkmacsystemfont, 'Corbel', 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', verdana, 'Verdana Ref', sans-serif;
 
-  --fontSizeSmall: 12px;
-  --fontSizeDefault: 14px;
-  --fontSizeBaseText: 16px;
   --iconSizeTiny: 16px;
-  --fontSizeLarge: 18px;
   --iconSizeSmall: 20px;
   --iconSizeDefault: 24px;
   --iconSizeBig: 176px;
 
+  --font-size-base: 16px;
```

Comment:
some of these variables have the same value, but I would like to keep it this way in case we ever want them to be different, so the variable name is more important than the value in other words.
Ex: button and subtitle 2 are the same, but in the future that may not be true, but button and subtitle will continue to exist regardless